COCOAPODS.md

```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
+           }
+
+           pod("CocoaLumberjack") {
+               source = git("https://github.com/CocoaLumberjack/CocoaLumberjack.git") {
+                   commit = "3e7f595e3a459c39b917aacf9856cd2a48c4dbf3"
+               }
+           }
+       }
+   }
+   ```
+   
+   </div>
+
+   > You can combine `branch`, `commit`, and `tag` parameters to get the specific version of a Pod.
+   {:.note}
+
+3. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Add a dependency on a Pod library from an archive
+
+You can add dependencies on a Pod library from `zip`, `tar`, or `jar` archive with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of a Pod library in the `pod()` function.
+In the configuration block specify the path to the archive: use the `url()` function with an arbitrary HTTP address in the `source` parameter value. 
+
+    Additionally, you can specify the Boolean `flatten` parameter as a second argument for the `url()` function.
+    It indicates that all Pod files located in the root directory of the archive.
+
+2. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("pod_dependency") {
+              source = url("https://github.com/Kotlin/kotlin-with-cocoapods-sample/raw/cocoapods-zip/cocoapodSourcesZip.zip", flatten = true)
+           }
+
+       }
+   }
+   ```
+   
+   </div>
+
+3. Re-import the project.
+
+> To work correctly with Xcode, you should specify the path to the Podspec in your Podfile.
+> For example:
+>
+> <div class="sample" markdown="1" theme="idea" data-highlight-only>
+>
+> ```ruby
+> target 'ios-app' do
+>     # ... other pod depedencies ...
+>    pod 'podspecWithFilesExample', :path => '../kotlin-libr/build/cocoapods/externalSources/url/podspecWithFilesExample'
+> end
+> ```
+> 
+> </div>
+>
+{:.note} 
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
```

Comment:
The link might be more specific.
Same for other sample links in the document.

Reply:
Do you mean links to the specific code lines? I'm not sure, because we talking about the whole project, not the 'usage example'.